File: isPalindrome/is_palindrome.py
def is_palindrome_1(s: str) -> bool:
    return s == s[::-1] 

print(is_palindrome_1('aaa'))
print(is_palindrome_1('abc'))

def is_palindrome_2(s: str) -> bool:
    for i in range(int(len(s) / 2)):
        if s[i] != s[len(s) - 1 - i]: return False
    return True


print(is_palindrome_2('aaa'))
print(is_palindrome_2('abc'))

File: isPalindrome/isPalindrome.js
/**
 * @param {string} str
 */
const isPalindrome1 = (str) => {
  const reverseStr = str.split('').reverse().join('');
  return reverseStr === str;
};

console.log(isPalindrome1('aaa'));
console.log(isPalindrome1('abc'));

/**
 * @param {string} str
 */
const isPalindrome2 = (str) => {
  for (let i = 0; i < str.length / 2; i++)
    if (str[i] !== str[str.length - 1 - i]) return false;
  return true;
};

console.log(isPalindrome2('aaa'));
console.log(isPalindrome2('abc'));
